2022/_data/menu.yml

```diff
@@ -22,6 +22,8 @@ items:
       href: "/diversity.html"
     - label: Code of conduct
       href: "/coc.html"
+- label: Venue
+  href: "/java-barcelona-venue.html"
```

Comment:
Creo que esta sección estaría duplicada 😉
![image](https://user-images.githubusercontent.com/2370634/169261685-4b7ed5ee-a381-4dab-8608-d5ef85b4ac52.png)

Reply:
resolved in #1153 thanks!